bokeh/sampledata/us_counties.py

```diff
@@ -20,7 +21,15 @@
 nan = float('NaN')
 
 data = {}
-with open(join(data_dir, 'US_Counties.csv')) as f:
+
+# csv differs in Python 2.x and Python 3.x. Open the file differently in each.
+filename = join(data_dir, 'US_Counties.csv')
+if sys.version_info[0] < 3:
+    f = open(filename, 'rb')
+else:
+    f = open(filename, 'r', newline='', encoding='utf8')
+
```

Comment:
@mattpap, I thought about that. The difference in opening r vs rb though and the other settings made me pause on that. Do you have an example of what you are thinking? Truthfully this seemed obvious to me what I was doing and I didn't try to further reduce.